.github/pull_request_template.md

```diff
@@ -1,49 +1,66 @@
 ## Motivation
-
 <!--
 Thank you for your Pull Request.
-How does this change improve Zebra?
 -->
+_What are the most important goals of the ticket or PR?_
+
+
+### PR Author Checklist
+
+Check before marking the PR as ready for review:
+  - [ ] Will the PR name make sense to users?
+    - [ ] Does it need extra CHANGELOG info?
+    - [ ] Does the PR have a priority label?
+  - [ ] Have you added or updated tests?
+  - [ ] Is the documentation up to date?
 
 ### Specifications
 
 <!--
 If this PR changes consensus rules, quote them, and link to the Zcash spec or ZIP:
 https://zips.z.cash/#nu5-zips
-If this PR changes network behaviour, quote and link to the Bitcoin network reference:
-https://developer.bitcoin.org/reference/p2p_networking.html
```

Comment:
We don't really do this any more, and we aren't making network protocol changes regularly. If we do, we can make a network ticket template.